.gitlab-ci.yml

```diff
@@ -0,0 +1,23 @@
+include: https://gitlab-templates.ddbuild.io/compute-delivery/v1/compute-delivery.yml
+
+stages:
+- verify
+- release
+- notify
+
+.ci:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/compute-delivery:latest
+  stage: verify
+  tags: ["runner:main"]
+
+build:
+  extends: .ci
+  script:
+    - GITHUB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name github.user.datadog-compute-robot --with-decryption --query "Parameter.{Value:Value}" --out text)
+    - git config --global url."https://${GITHUB_TOKEN}@github.com/DataDog".insteadOf https://github.com/DataDog
+    - go get github.com/onsi/ginkgo/ginkgo
+    - go get github.com/containernetworking/cni/cnitool
+    - go get golang.org/x/tools/cmd/cover
+    - go get github.com/modocache/gover
+    - go get github.com/mattn/goveralls
+    - ./build_linux.sh
```

Comment:
I commented [here](https://github.com/DataDog/plugins/pull/1#pullrequestreview-378858170) about running `./test_linux.sh` but basically we don't have the necessary permissions inside a Docker container for running tests.

I also tried to run `scripts/release.sh` but that also wasn't fruitful because that requires running a Docker container. I suspect that we could actually make this work if we wanted to but I'm not sure it's worth our time at this point. I'd rather wait to try to get some feedback from upstream about this work and internally I'll [build/release an RC on Monday](https://github.com/containernetworking/plugins/blob/master/RELEASING.md) for the `dummy` plugin so we can get that deployed to a cluster for testing.